src/components/calcite-input-date-picker/calcite-input-date-picker.e2e.ts

```diff
@@ -48,6 +48,7 @@ describe("calcite-input-date-picker", () => {
     const date = await page.find("calcite-input-date-picker");
 
     await date.click();
+    await page.waitForChanges();
```

Comment:
I think this might need to go before the click. I'm not sure why the test couldn't find the date but it failed on clicking.

`Node is either not visible or not an HTMLElement`

Reply:
Something else is tripping this up then. To my knowledge, `find` does not trigger any page changes. Digging...

Reply:
yeah it seems like this shouldn't ever fail :\

Reply:
Now it's failing on a combobox test. 😂

Reply:
<img src="https://media.giphy.com/media/gd09Y2Ptu7gsiPVUrv/giphy.gif" />